py-plumed/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "py-plumed" %}
+{% set version = "2.10" %}
```

Comment:
Is there a standard way to make sure the date is baked in the version in some way? What I am thinking about is if a few weeks we release 2.10 it should override this one. Or shall we just increase the build: number below?

Reply:
Ok, I changed this to have the version be `2.10.dev1` and store the git sha in the build string instead. The whole package is then `plumed-2.10.dev1-git.b0ffcb55ad8cc3c71ec5c30a1bd46f68dd33bceb`.

This means that when 2.10 is released, it will be used instead of this one, since conda is following PEP440: https://peps.python.org/pep-0440/